integration/pmem/pmem_test.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+dir_path=$(dirname "$0")
+source "${dir_path}/../../lib/common.bash"
+source "${dir_path}/../../.ci/lib.sh"
+image="fedora"
+payload="tail -f /dev/null"
+container_name="test-pmem"
+osbuilder_repository="github.com/kata-containers/osbuilder"
+osbuilder_repository_path="${GOPATH}/src/${osbuilder_repository}"
+test_directory_name="test_pmem1"
+test_directory="/tmp/${test_directory_name}"
+
+function setup() {
+	clean_env
+	check_processes
+	if [ ! -d "${osbuilder_repository_path}" ]; then
+		go get -d "${osbuilder_repository}" || true
+	fi
+}
+
+function test_pmem {
+	sudo mkdir "${test_directory}"
+	pushd "${test_directory}"
+
+	# Create xfs
+	sudo dd if=/dev/zero of=xfs.img bs=1M count=128
+	device_name=$(sudo losetup --offset 2M --show -Pf xfs.img)
+	sudo mkfs.xfs "${device_name}"
+	popd
+
+	pushd "${osbuilder_repository_path}/image-builder"
+	size="2097152"
+	gcc nsdax.gpl.c -o nsdax
+	sudo ./nsdax "${test_directory}/xfs.img" "${size}" "${size}"
+	popd
+
+	pushd "${test_directory}"
+	sudo mount "${device_name}" "${test_directory}"
+
+	# Running container
+	docker run -d --name "${container_name}" --runtime kata-runtime -v "${test_directory}:/${test_directory_name}" "${image}" sh -c "${payload}"
+
+	# Check container
+	docker exec "${container_name}" sh -c "mount | grep ${test_directory_name} | grep '/dev/pmem'"
+	docker exec "${container_name}" sh -c "mount | grep ${test_directory_name} | grep 'dax'"
```

Comment:
please check dax in the above line: `docker exec "${container_name}" sh -c "mount | grep ${test_directory_name} | grep '/dev/pmem' | grep 'dax'"`